test/scripts/publish_message.py

```diff
@@ -20,8 +20,8 @@
     data = json.dumps({
         "timeCreated": "2008-08-24T00:00:00Z",
         "metadata": {
-            "case_id": case_id,
-            "tx_id": tx_id,
+            "tx_id": "123456789",
```

Comment:
If we're no longer using the generated UUID's, lines 18-19 can be removed.

Reply:
good spot, should still be a uuid

Reply:
fixed